nullaway/src/main/java/com/uber/nullaway/generics/GenericsChecks.java

```diff
@@ -715,6 +785,40 @@ public static Nullness getGenericReturnNullnessAtInvocation(
       MethodInvocationTree tree,
       VisitorState state,
       Config config) {
+
```

Comment:
We need to update the Javadoc of `getGenericReturnNullnessAtInvocation` to indicate it also handles generic methods, in addition to instance methods of generic classes (what it handled before)